src/main/java/com/dougnoel/sentinel/steps/TableVerificationSteps.java

```diff
@@ -275,4 +275,14 @@ public static void verifyColumnIsEmpty(String columnName, String tableName, Stri
         }
     }
 
+    @Then("^I verify the (.*?) column in the (.*?) is displayed to the left of the (.*?) column$")
+    public static void verifyColumnDisplayOrder(String column1Name, String tableName, String column2Name){
+        var expectedResult = SentinelStringUtils.format(
+                "Expected the {} column of the {} to be displayed to the left of the {} column.",
+                column1Name, tableName, column2Name);
+        log.trace(expectedResult);
+
+        assertTrue(expectedResult, getElementAsTable(tableName).verifyColumnDisplayOrder(column1Name, column2Name));
```

Comment:
Yeah just curious if the table implementation is truly better than just doing it in here as it's so specific of a scenario.

Reply:
Also if we're doing this step we can easily extend this to do a right or left check. Might be useful for later.

Reply:
...Point taken!